docs/SPECIFICATION.md

```diff
@@ -144,7 +144,7 @@ message MessageData {
 A MessageData `data` in a Message `m` must pass the following validations:
 
 1. `m.data.type` must be a valid MessageType.
-2. `m.data.fid` must be an integer >= 0.
+2. `m.data.fid` must be an integer > 0.
```

Comment:
⚠️  calling this out: this is consistent with CastId defining fid as > 0. If the intention here is that fid can be explicitly 0, I think it should be stated otherwise it just looks inconsistent. 

Also: does it mean that any fid is valid even if they are not in the Id registry?